src/core/Mixer.cpp

```diff
@@ -224,6 +224,13 @@ int Mixer::createChannel()
 	// reset channel state
 	clearChannel( index );
 
+	// if there is a soloed channel, mute the new track
+	if (m_lastSoloed != -1 && m_mixerChannels[m_lastSoloed]->m_soloModel.value())
+	{
+		m_mixerChannels[index]->m_muteBeforeSolo = m_mixerChannels[index]->m_muteModel.value();
```

Comment:
I think `m_mixerChannels[index]->m_muteModel.value()` will always be `false` unless we put something between `clearChannel(index)` and here.

Reply:
Yes, that's true. But because it's set on the function clearChannel I wanted to use its value rather than presume it's false (even though we now is false) But maybe I'm overthinking things ;-)
If you think it's clearer/better to set it to false directly, I can change the code. No problem.

Reply:
If you have such an intention, no problem.